rpc/.js/src/main/scala/io/udash/rpc/internals/ServerConnector.scala

```diff
@@ -68,29 +65,20 @@ abstract class AtmosphereServerConnector[RPCRequest](
     Atmosphere.subscribe(atmRequest)
   }
 
-  override def sendRPCRequest(request: RPCRequest): Unit = {
-    val msg = requestToString(request)
+  override def sendRpcRequest(request: RpcRequest): Unit = {
+    val msg = JsonStringOutput.write[RpcRequest](request)
     if (isReady == ConnectionStatus.Open) socket.push(msg)
     else waitingRequests += request
   }
 
-  private def handleMessage(msg: String) = {
-    import localFramework.RPCRequestCodec
-    import remoteFramework.RPCResponse
+  private def handleMessage(msg: String): Unit = {
     implicit val ecr: ExceptionCodecRegistry = exceptionsRegistry
-    try handleResponse(remoteFramework.read[RPCResponse](JsonStr(msg))) catch {
-      case _: Exception =>
-        try {
-          localFramework.read[localFramework.RPCRequest](JsonStr(msg)) match {
-            case fire: localFramework.RPCFire =>
-              handleRpcFire(fire)
-            case unhandled =>
-              logger.error(s"Unhandled RPCRequest: $unhandled")
-          }
-        } catch {
-          case _: Exception =>
-            logger.error(s"Unhandled message: $msg")
-        }
+    try JsonStringInput.read[RpcServerMessage](msg) match {
```

Comment:
is this new behavior tested?

Reply:
It's a change from https://github.com/UdashFramework/udash-core/pull/222, but I believe it is tested. :)